File: dev/drinks/drinks.cpp
#include <bits/stdc++.h>

auto main() -> int {
  int n; int x; std::vector<int> d;
  std::cin >> n;
  while (std::cin >> x) {
    d.push_back(x);
  }
  double vol = 0;
  for (auto& i : d) {
    vol += i;
  }

  double res = ((vol / (n * 100)) * 100);
  std::cout << res << '\n';
}

File: dev/drinks/drinks.py
#!/usr/bin/env python3

import sys
lines = sys.stdin.read().splitlines()

n = int(lines[0])
drinks = list(map(int, lines[1].split()))

vol = 0
for x in range(n):
    vol += drinks[x]

res = (vol / (n * 100)) * 100

print(res)
